src/spring-cloud/azext_spring_cloud/commands.py

```diff
@@ -81,5 +86,18 @@ def load_command_table(self, _):
         g.custom_command('redis update', 'binding_redis_update')
         g.custom_show_command('remove', 'binding_remove')
 
+    with self.command_group('spring-cloud certificate', client_factory=cf_certificates) as g:
+        g.custom_command('add', 'certificate_add')
+        g.custom_command('show', 'certificate_show', table_transformer=transform_spring_cloud_certificate_output)
+        g.custom_command('list', 'certificate_list', table_transformer=transform_spring_cloud_certificate_output)
+        g.custom_command('remove', 'certificate_remove')
+
+    with self.command_group('spring-cloud app custom-domain', client_factory=cf_custom_domains) as g:
+        g.custom_command('bind', 'domain_bind')
+        g.custom_command('show', 'domain_show', table_transformer=transform_spring_cloud_custom_domain_output)
+        g.custom_command('list', 'domain_list', table_transformer=transform_spring_cloud_custom_domain_output)
+        g.custom_command('update', 'domain_update')
+        g.custom_command('unbind', 'domain_unbind')

```

Comment:
In `_help.py`, it's called `delete`. Please make it consistent and run `azdev linter spring-cloud` to validate.